grammars/css.cson

```diff
@@ -561,63 +561,113 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (:)
+          (
+            link|target|visited                                     # location
+          | active|focus|hover                                      # user action
+          | default|disabled|enabled|read-only|read-write           # input control state
+          | checked|indeterminate                                   # input value state
```

Comment:
Looks like we lose the extra `.ui-state` class here.  Do you think that's important?